rosie/chamber_of_deputies/classifiers/election_expenses_classifier.py

```diff
@@ -14,11 +14,11 @@ class ElectionExpensesClassifier(TransformerMixin):
         Brazilian Federal Revenue category of companies, preceded by its code.
     """
 
-    def fit(self, X):
+    def fit(self, dataset):
         return self
 
-    def transform(self, X=None):
+    def transform(self, dataset=None):
         return self
 
-    def predict(self, X):
-        return X['legal_entity'] == '409-0 - CANDIDATO A CARGO POLITICO ELETIVO'
+    def predict(self, dataset):
+        return dataset['legal_entity'] == '409-0 - CANDIDATO A CARGO POLITICO ELETIVO'
```

Comment:
Hey @cuducos @anaschwendler! Any tip how could I give a more meaningful name for `409-0 - CANDIDATO A CARGO POLITICO ELETIVO`?

Reply:
I'd make of it a class constant, e.g.:

```python
class ElectionExpensesClassifier:

    ELECTION_LEGAL_ENTITY = '409 - …'
```

Reply:
The answer for that questions is here: https://concla.ibge.gov.br/estrutura/natjur-estrutura/natureza-juridica-2014/409-0-candidato-a-cargo-politico-eletivo.html

I think that going for constant is the best, because we don't know if we may use another code from that dataset

Reply:
> The answer for that questions is here: https://concla.ibge.gov.br/estrutura/natjur-estrutura/natureza-juridica-2014/409-0-candidato-a-cargo-politico-eletivo.html

I'm sorry but this does not enough context to give name for `409-0` and `CANDIDATO A CARGO POLITICO ELETIVO`. I even know if this is really splitted stuffs that could receive different names? May that is the reason why it's strange to me. I just saw that the serie returned has two boolean values what make thing that it's one for each pieces of the constant.

> I think that going for constant is the best, because we don't know if we may use another code from that dataset

I agree. I introduced it here: https://github.com/datasciencebr/rosie/pull/89/commits/acbb6c2a06ccb7803288cae20db00094a95a91e6#diff-69f39fa56efc440a1649e8ba9ff1f1cbR24 as suggested.